cvat/apps/engine/annotation.py

```diff
@@ -1227,13 +1227,6 @@ def init_from_db(self):
             overlap = self.db_task.overlap
             self._merge_data(annotation.data, start_frame, overlap)
 
-    @staticmethod
```

Comment:
We should not delete the code for now. Otherwise old tasks will be dumped incorrectly.

Reply:
We should to write a migration file which rotates all coordinates.

Reply:
OK. Will you do that?

Reply:
Yes, I will.